path_credentials.go

```diff
@@ -103,8 +103,12 @@ func (b *tfBackend) createUserCreds(ctx context.Context, req *logical.Request, r
 		"token_id": token.ID,
 	}
 
-	if role.Description != "" {
-		data["description"] = role.Description
+	if token.Description != "" {
+		data["description"] = token.Description
+	}
```

Comment:
For the other token types, will changing from `role.Description` to `token.Description` remain correct?

Reply:
the description from token will contain the randomness generated alongside the token and include it in the cred output